content/blog/asyncapi_codegen_scst.md

```diff
@@ -0,0 +1,136 @@
+---
```

Comment:
please add:
```
type: blog
featured: true
```

so Hugo knows it is blog and that it should be featured on top

Reply:
fixed